### PR TITLE
HARVESTER: Update Chinese translation

### PR DESCRIPTION
**Pull request by vickyhella**
- Update Chinese translation for Harvester

### CODE REVIEW

**Comment by vickyhella** [comment]
@WuJun2016 Please take a look and thanks!!

**Review by WuJun2016 — COMMENTED** [review]
@guangbochen   PTAL,  
other LGTM

File: pkg/harvester/l10n/zh-hans.yaml [hunk 1]
```diff
@@ -297,6 +300,7 @@ harvester:
       storage: 存储
       labels: 标签
       ksmtuned: Ksmtuned
+      seeder: 带外访问
```
[the remark this answers is not in the document]
Comment (guangbochen) [reply pkg/harvester/l10n/zh-hans.yaml:303]: [seeder ](https://github.com/harvester/seeder) is a project name; so you don't need to translate it.

File: pkg/harvester/l10n/zh-hans.yaml [hunk 2]
```diff
@@ -999,15 +1022,39 @@ harvester:
   secret:
     label: 密文
 
+  sriov:
+    label: SR-IOV 网络设备
+    nodeName: 节点
+    numVFs: 虚拟函数数量
```
[the remark this answers is not in the document]
Comment (guangbochen) [reply pkg/harvester/l10n/zh-hans.yaml:1032]: Why are we using `numVFs` here, shouldn't the UI use full words? in regards to the translation, prefer to use `虚拟数量`

File: pkg/harvester/l10n/zh-hans.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (vickyhella) [reply pkg/harvester/l10n/zh-hans.yaml:304]: The English file uses `Out-of-Band access` here. Should I also change it to `Seeder` as well? @guangbochen

File: pkg/harvester/l10n/zh-hans.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (WuJun2016) [reply pkg/harvester/l10n/zh-hans.yaml:1032]: @DaiYuzeng   Please keep the text in the table and the dialog consistent.

[the remark this answers is not in the document]
Comment (DaiYuzeng) [reply pkg/harvester/l10n/zh-hans.yaml:1032]: https://github.com/vickyhella/dashboard/pull/2

@vickyhella I would like to send a PR into your repo: https://github.com/vickyhella/dashboard/pull/2

The column label of sr-iov VFs wil be `Number of virtual functions` now. @WuJun2016

[the remark this answers is not in the document]
Comment (vickyhella) [reply pkg/harvester/l10n/zh-hans.yaml]: @DaiYuzeng's PR is merged and the translation is updated to `虚拟数量`.

**Comment by vickyhella** [comment]
@WuJun2016 already rebased and updated the translations according to comments (and also added several new translations from the master head). Please take a second look and thanks!!

**Review by WuJun2016 — APPROVED** [review]
LGTM, thanks

I will modify the `seeder` key after this PR is merged.

**Pull request merged**